frontend/packages/dev-console/src/components/catalog/catalog-plugin.ts

```diff
@@ -15,7 +15,7 @@ export const catalogPlugin: Plugin<CatalogConsumedExtensions> = [
         '%devconsole~Browse for Container Images that support a particular language or framework. Cluster administrators can customize the content made available in the catalog.%',
       // t('devconsole~**Builder Images** are Container Images that build source code for a  particular language or framework.')
       typeDescription:
-        '%devconsole~**Builder Images** are container Images that build source code for a  particular language or framework.%',
+        '%devconsole~**Builder Images** are Container Images that build source code for a  particular language or framework.%',
```

Comment:
It seems a little odd to me that "Container Image" is capitalized since it make it sound like a k8s resource when it's not. If we do it here, there are other places in the UI we need to update as well to be consistent. (I have a preference for "container image," though.)

```
$ rg 'container image' -c
packages/container-security/src/components/summary.tsx:1
packages/container-security/locales/zh/container-security.json:1
packages/container-security/locales/en/container-security.json:1
packages/container-security/locales/ja/container-security.json:1
packages/console-shared/src/utils/sample-utils.ts:1
packages/dev-console/src/components/catalog/catalog-plugin.ts:4
packages/dev-console/integration-tests/features/Knative/create-knative-workload.feature:1
packages/dev-console/locales/en/devconsole.json:2
packages/dev-console/src/utils/__tests__/add-resources-menu-utils.spec.tsx:1
packages/dev-console/src/utils/__tests__/imagestream-test-data.ts:2
packages/dev-console/integration-tests/features/addFlow/create-from-container-image.feature:2
packages/operator-lifecycle-manager/src/components/create-catalog-source.tsx:2
packages/operator-lifecycle-manager/src/components/descriptors/reference/reference.md:2
packages/operator-lifecycle-manager/locales/zh/create-catalog-source.json:2
packages/operator-lifecycle-manager/locales/en/create-catalog-source.json:2
packages/operator-lifecycle-manager/locales/ja/create-catalog-source.json:2
public/module/k8s/container.ts:3
packages/kubevirt-plugin/src/components/create-vm-wizard/tabs/vm-settings-tab/provision-source.tsx:1
packages/kubevirt-plugin/locales/en/kubevirt-plugin.json:1
__tests__/components/pod.spec.tsx:1
```